lib/expo/message.ex

```diff
@@ -15,8 +15,8 @@ defmodule Expo.Message do
 
   alias Expo.Message.{Plural, Singular}
 
-  @type msgid :: [String.t()]
-  @type msgstr :: [String.t()]
+  @type msgid :: nonempty_list(String.t())
+  @type msgstr :: nonempty_list(String.t())
```

Comment:
You can use `...` to keep the list syntax if you want:

```suggestion
  @type msgid :: [String.t(), ...]
  @type msgstr :: [String.t(), ...]
```

Reply:
@whatyouhide Cool, didn't know that :)